src/modular/boxed_residue/add.rs

```diff
@@ -63,6 +63,7 @@ impl AddAssign<BoxedResidue> for BoxedResidue {
 }
 
 #[cfg(test)]
+#[allow(clippy::unwrap_used)]
```

Comment:
This seems superfluous (as do a few others)

Reply:
It's not though, there's `unwrap()` right below in `add_overflow()`.

Reply:
It passes for me locally as well as in CI. Do you have some local configuration overridden that checks for `unwrap()` in tests?

Reply:
I'm using Rust 1.73, and CI uses 1.72, perhaps that's the reason?

Reply:
Ah also CI doesn't have `--tests` in the `clippy` call

Reply:
```
$ rustc --version
rustc 1.74.0 (79e9716c9 2023-11-13)
$ cargo clippy
    Finished dev [unoptimized + debuginfo] target(s) in 0.01s
```

Reply:
If I add `--tests` I get several warnings about the tests, but we usually don't run clippy on tests

Reply:
Ok, no problem, I usually do. These clippy fixes are all in the tests, should I roll them back?

Reply:
Yeah (or at least file it in a separate PR)

Reply:
Rolled back.